### DISCUSSION TITLE
Toml remapings does not work

### DISCUSSION THREAD

**Discussion by NicolaMirchev** [Cyfrin/foundry-full-course-cu#223] [Q&A] [answered]
Hey, guys.
I am importing  "import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol". I have installed it locally and have put in my toml file "remapings = ["@chainlink/contracts/=lib/chainlink-brownie-contracts/contracts/"]".
Unfortunately when I try to run the code I am getting "Source "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" not found: File not found. Searched the following locations: "/home/nikola_mirchev8/solidity-course/foundry-fund-me-f23". 
foundry-fund-me-f23 is the root folder for the project.
Do you have any suggestions what could be the problem? Thanks!

**Comment by Shiva-Sai-ssb** [comment]
Hey @NicolaMirchev, have you saved the foundry.toml file after adding the remappings line?

**Comment by pyromaus** [comment, marked as answer]
add `p` to `remappings`

**Reply by Shiva-Sai-ssb** [reply]
Indeed @pyromaus, there appears to be a typo error. Hey @NicolaMirchev, the correct term is "remappings" instead of "remapings."

**Reply by NicolaMirchev** [reply]
Ahh... those errors... 😠
Thank you! Everything works now!

**Reply by cromewar** [reply]
Amazing @pyromaus

**Comment by sharonphiliplima** [comment]
I am getting this OS error guys
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/69852010/5193401d-ae83-4605-8e61-eb5a844a5ceb)

**Reply by ChefAharoni** [reply]
@sharonphiliplima 
Remappings can be tricky, foundry is a little bit buggy in VSCode.

Try these [solutions](https://github.com/Cyfrin/foundry-full-course-f23/discussions/194) regarding mappings issues.

For me, writing the remappings in a .txt file helped, this is how it looks in mine:
<img width="378" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/4399057/dda8f167-0918-4206-8ea1-538e980e49ed">

```text
@openzeppelin/=lib/openzeppelin-contracts/
```

You can also run `forge remappings` to see the remappings foundry is recognizing.
